### PR TITLE
feat: Add new landing page template with dynamicly generated pages

### PR DESCRIPTION
**Pull request by jperasmus**

### CODE REVIEW

File: .github/workflows/deploy-website.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ jobs:
           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
 
       - name: Checkout Repo
-        uses: actions/checkout@v2
+        uses: actions/checkout@v1
```
Comment (caweidmann) [inline .github/workflows/deploy-website.yml]: Why the change to v1?

Comment (jperasmus) [reply .github/workflows/deploy-website.yml]: v2 does only a shallow (1 level deep checkout), so it can't be used to determine if files changed for specific rebuilds if that happened more than one commit ago.
